.github/workflows/aws-deployment.yml

```diff
@@ -89,10 +91,13 @@ jobs:
           name: aws-deploy-files
           path: ./packages/web/dist
 
-      # Note that the command below will copy the files to the root of the S3 bucket e.g., s3://biofile-finder.allencell.org/
-      # If you want to copy files to a S3 prefix / subdirectory, you would want something like ${{ env.S3_BUCKET }}/your_prefix below
-      - name: Copy build files to S3 root
-        run: aws s3 sync ./packages/web/dist ${{ env.S3_BUCKET }}
+      # Note that the command below will copy the files to the root of the S3 bucket e.g., s3://bff.allencell.org/
+      # If you want to copy files to a S3 prefix / subdirectory, you would want something like ${{ env.BFF_S3_BUCKET }}/your_prefix below
+      - name: Copy build filed to BFF S3
+        run: aws s3 sync ./packages/web/dist ${{ env.BFF_S3_BUCKET }}
 
-      - name: Invalidate CloudFront cache
+      - name: Invalidate CloudFront biofile-finder cache
```

Comment:
Maybe add a comment that this isn't strictly necessary since we aren't writing anything served directly by this distribution, since all it does is issue redirects.